client/src/components/Messaging/NewMessageForm.js

```diff
@@ -29,8 +38,24 @@ export const NewMessageForm = ({ handleSubmit, text, setText }) => {
       />
       <div className={styles.imagePickerContainer}>
```

Comment:
I would suggest making a separate component for imagePicker - which will pick and show preview